### PR TITLE
feat(icons): add holdout icon

### PR DESCRIPTION
**Pull request by cmaija**
## Summary

This PR adds the new icon 'holdout'. It will be used to represent a holdout object in the experimentation product. 

## Screenshots (if appropriate):

holdout-icon

## Testing approaches

- ensure existing tests pass and visual check in storybook

### CODE REVIEW

File: packages/icons/src/img/sprite.svg [hunk 1]
```diff
@@ -1983,5 +1983,10 @@
         clip-rule="evenodd"
       />
     </symbol>
+    <symbol viewBox="0 0 20 20" id="lp-icon-holdout">
```
Comment (pheggeseth) [inline packages/icons/src/img/sprite.svg:1986]: It would be awesome if you'd be willing to put this icon in alphabetical order like the rest of them! 🙏🏻

**Review by pheggeseth — APPROVED** [review]
LGTM other than moving the symbol in the sprite file to be in alphabetical order, though I don't have any context as to whether the icon itself is correct.

**Review by matthewferry — APPROVED** [review]
Some nits
- move holdout symbol to alphabetical
- each attribute should be on its own line if there are more than 1

The svg path looks to be just a touch off-center. not a huge deal, but ideally you can swap the path definition for:

```
<symbol viewBox="0 0 20 20" id="lp-icon-holdout">
  <path
    fill-rule="evenodd"
    d="M13.75 9.984a1.15 1.15 0 1 0 0 2.3 1.15 1.15 0 0 0 0-2.3Zm-2.65 1.15a2.65 2.65 0 1 1 5.3 0 2.65 2.65 0 0 1-5.3 0Z"
    clip-rule="evenodd"
  />
  <path d="M9.917 14.067a2.4 2.4 0 1 1-4.8 0 2.4 2.4 0 0 1 4.8 0Z"/>
  <path
    fill-rule="evenodd"
    d="M6.75 4.984a1.65 1.65 0 1 0 0 3.3 1.65 1.65 0 0 0 0-3.3ZM3.6 6.634a3.15 3.15 0 1 1 6.3 0 3.15 3.15 0 0 1-6.3 0Z"
    clip-rule="evenodd"
  />
</symbol>

File: packages/icons/src/img/sprite.svg (hunk 1 above)
Comment (cmaija) [reply packages/icons/src/img/sprite.svg:1986]: Oh sorry about that, I will move into the correct order!

**Comment by cmaija** [comment]
@matthewferry @pheggeseth  Thanks so much yall for reviewing, I moved the svg up to the correct order and swapped the path definition ✅

**Pull request merged**